File: SearchAndSorting/LinearSearch.py
def search(arr,item):
    for i in range(max):
        if arr[i] == item:
            return i

    return -1

arr= [10,20,30,40,15]

max = len(arr)
desired_item = int(input("enter desired item : "))
result = search(arr,desired_item)

if result == -1:
    print("item not found")
else:
    print("item found on index : " + str(result))

File: SearchAndSorting/LinearSearch.c
#include<stdio.h>
#define max 5
int i;
int search(int arr[],int item){
    for(i=0;i<max;i++){
        if(arr[i]==item){
            return i;
        }
    }
    return -1;
}

void main(){
    int arr[max],item;
    printf("input numbers:");
    for(i=0;i<max;i++){
        scanf("%d",&arr[i]);
    }
    printf("\ninput search item: ");
    scanf("%d",&item);
    int result = search(arr,item);
    if (result == -1){
        printf("item not found");
    }
    else{
        printf("item found on %d index\n",result);
    }
}
